python/Ganga/Core/GangaRepository/VStreamer.py

```diff
@@ -361,7 +364,9 @@ def end_element(name):
                 # unescape the special characters
                 s = unescape(self.value_construct)
                 #logger.debug('string value: %s',s)
-                val = eval(s, config_scope)
+                if s not in _cached_eval_strings:
+                    _cached_eval_strings[s] = eval(s, getProxyInterface().__dict__)
+                val = copy.deepcopy(_cached_eval_strings[s])
```

Comment:
I'm not sure that we should have to. What is the string that is being evaluated that requires `datetime` to be present in the namespace? When processing the name of a `ComponentItem` in a `<class>` entry we use the plugin system and when processing an value in a `<value>` entry we use a standard python namespace (and at *that* point, yes we require `datetime` to be present).